lib/liquid/block_body.rb

```diff
@@ -76,6 +76,9 @@ def render(context)
           end
         rescue MemoryError => e
           raise e
+        rescue MissingVariable, MissingDropMethod, MissingFilter => e
+          context.handle_error(e, token.line_number)
+          output << nil
```

Comment:
We didn't want to print exception messages to the output. Actually this approach was implemented in https://github.com/Shopify/liquid/issues/490 so I also decided to use it.